commercetools-sdk-base/src/test/java/io/sphere/sdk/queries/PagedQueryResultTest.java

```diff
@@ -44,6 +46,32 @@ public void middleOfManyResult() throws Exception {
     }
 
     @Test
+    public void getResultPage() throws Exception {
+        final PagedQueryResult<Integer> queryResult = PagedQueryResult.of((long) 80, TOTAL, listOfSize(PAGE_SIZE));
+        assertThat(queryResult.getPage()).isEqualTo(3);
```

Comment:
or maybe just show the total page number which is the max, since it would be inconvenient to block the whole page from loading because of a small bug form the backend
i think that idea is much better.